src/components/Form.js

```diff
@@ -21,6 +22,7 @@ import OrganizationSelector from './form/OrganizationSelector';
 import { ScrollablePanel } from './elements/Scrollable';
 import { RgpdContact } from './form/RgpdContact';
 import { Contact } from './form/Contact';
+import ScopesForm from './form/ScopesForm';
```

Comment:
I do not think suffix-ing variable name with Form is necessary. `Scopes` is enough in my opinion (like `Contact`)